src/VirtualClient/VirtualClient.Actions/SysbenchOLTP/SysbenchOLTPClientExecutor.cs

```diff
@@ -263,22 +256,14 @@ private Task ExecuteWorkloadAsync(EventContext telemetryContext, CancellationTok
 
         private async Task InstallSysbenchOLTPPackage(CancellationToken cancellationToken)
         {
-            const string autogenScriptCommand = "./autogen.sh";
-            const string configureScriptCommand = "./configure";
-            const string makeCommand = "make -j";
-            const string makeInstallCommand = "make install";
-
-            // build sysbench
-            await this.ExecuteCommandAsync<SysbenchOLTPClientExecutor>(autogenScriptCommand, null, this.sysbenchDirectory, cancellationToken)
-                .ConfigureAwait(false);
+            const string updateAptCommand = "su -c \"bash <( curl -s https://packagecloud.io/install/repositories/akopytov/sysbench/script.deb.sh)\"";
+            const string installSysbenchCommand = "apt install sysbench";
```

Comment:
It's fine if we can only support ubuntu for now, but we need to research if there are scripts for other distros, and we need proper error message if we are not in the correct distro.